desktop/assets/texts/ja-JP/main_menu.ftl

```diff
@@ -37,4 +37,4 @@ debug-menu-open-movie-list = 既存の映像情報
 debug-menu-open-domain-list = ドメインを表示
 debug-menu-search-display-objects = 表示オブジェクトを検索...
 view-menu = 表示
-view-menu-fullscreen = フルスクリーン
+view-menu-fullscreen = 全画面表示
```

Comment:
That seems to be Chinese?

Reply:
No, I thought so too for the first time - the difference is that the current text is a transliteration (so it's pronounced similar to "fullscreen"), while the new suggestion is a proper (semantic) translation.